From: Stefan Dösinger <stefan@codeweavers.com>
Subject: Proposed fix for WineHQ bug #28201
Last-Update: 2012-05-19
Origin: upstream, http://bugs.winehq.org/attachment.cgi?id=39565
Bug: http://bugs.winehq.org/show_bug.cgi?id=28201
---
 dlls/wined3d/device.c |    5 ++---
 1 file changed, 2 insertions(+), 3 deletions(-)

--- a/dlls/wined3d/device.c
+++ b/dlls/wined3d/device.c
@@ -5264,6 +5264,7 @@ HRESULT CDECL wined3d_device_reset(struc
         wined3d_surface_decref(device->onscreen_depth_stencil);
         device->onscreen_depth_stencil = NULL;
     }
+    wined3d_device_set_depth_stencil(device, NULL);
 
     LIST_FOR_EACH_ENTRY_SAFE(resource, cursor, &device->resources, struct wined3d_resource, resource_list_entry)
     {
@@ -5363,11 +5364,9 @@ HRESULT CDECL wined3d_device_reset(struc
         device->onscreen_depth_stencil = NULL;
     }
 
-    /* Reset the depth stencil */
+    /* Apply the auto depth stencil if the app requested one */
     if (swapchain_desc->enable_auto_depth_stencil)
         wined3d_device_set_depth_stencil(device, device->auto_depth_stencil);
-    else
-        wined3d_device_set_depth_stencil(device, NULL);
 
     TRACE("Resetting stateblock\n");
     wined3d_stateblock_decref(device->updateStateBlock);
