Do not use this patch
Index: vtk-5.8.0/Hybrid/CMakeLists.txt
===================================================================
--- vtk-5.8.0.orig/Hybrid/CMakeLists.txt	2011-09-01 15:48:13.000000000 +0200
+++ vtk-5.8.0/Hybrid/CMakeLists.txt	2011-09-01 15:49:47.000000000 +0200
@@ -26,7 +26,7 @@
 ENDIF (WIN32)
 
 # for vtkImageDataLIC2D and vtkStructuredGridLIC2D
-SET( KIT_LIBS ${KIT_LIBS} vtkftgl )
+SET( KIT_LIBS ${KIT_LIBS} ftgl )
 
 SET ( Kit_SRCS
 vtk3DSImporter.cxx
Index: vtk-5.8.0/Rendering/CMakeLists.txt
===================================================================
--- vtk-5.8.0.orig/Rendering/CMakeLists.txt	2011-09-01 15:49:34.000000000 +0200
+++ vtk-5.8.0/Rendering/CMakeLists.txt	2011-09-01 15:49:47.000000000 +0200
@@ -9,7 +9,7 @@
 ENDIF (JAVA_AWT_LIBRARY)
 
 SET(KIT_INTERFACE_LIBRARIES vtkGraphics vtkImaging)
-SET(KIT_LIBS vtkIO vtkftgl)
+SET(KIT_LIBS vtkIO ftgl)
 #INCLUDE(${VTK_CMAKE_DIR}/vtkTestGL.cmake)
 #INCLUDE(${VTK_CMAKE_DIR}/vtkTestGLX.cmake)
 
Index: vtk-5.8.0/Rendering/vtkFreeTypeUtilities.cxx
===================================================================
--- vtk-5.8.0.orig/Rendering/vtkFreeTypeUtilities.cxx	2011-09-01 15:48:13.000000000 +0200
+++ vtk-5.8.0/Rendering/vtkFreeTypeUtilities.cxx	2011-09-01 15:49:47.000000000 +0200
@@ -23,9 +23,10 @@
 
 // FTGL
 
-#include "vtkftglConfig.h"
-#include "FTLibrary.h"
-#include "FTGLPixmapFont.h"
+//#include "vtkftglConfig.h"
+//#include "FTLibrary.h"
+//#include "FTGLPixmapFont.h"
+#include <FTGL/ftgl.h>
 
 // The embedded fonts
 
@@ -81,7 +82,7 @@
 #if VTK_FTFC_DEBUG_CD
   printf("vtkFreeTypeUtilitiesCleanup::vtkFreeTypeUtilitiesCleanup\n");
 #endif
-  FTLibraryCleanup::AddDependency(&vtkFreeTypeUtilitiesCleanupCallback);
+  //FTLibraryCleanup::AddDependency(&vtkFreeTypeUtilitiesCleanupCallback);
 }
 
 //----------------------------------------------------------------------------
@@ -181,11 +182,11 @@
   printf("vtkFreeTypeUtilities::GetLibrary\n");
 #endif
 
-  FTLibrary *ftgl_lib = FTLibrary::GetInstance();
-  if (ftgl_lib)
-    {
-    return ftgl_lib->GetLibrary();
-    }
+  //FTLibrary *ftgl_lib = FTLibrary::GetInstance();
+  //if (ftgl_lib)
+  //  {
+  //  return ftgl_lib->GetLibrary();
+  //  }
 
   return NULL;
 }
@@ -1513,7 +1514,7 @@
 
   // OK the font is not cached, try to create one
 
-  FTFont *font = new FTGLPixmapFont;
+  FTFont *font = NULL; // = new FTGLPixmapFont;
 
   // A face file name has been provided, try to load it, otherwise
   // just use the embedded fonts (i.e. font family, bold and italic attribs)
@@ -1618,7 +1619,8 @@
     unsigned char *ptr = EmbeddedFonts
       [tprop->GetFontFamily()][tprop->GetBold()][tprop->GetItalic()].ptr;
 
-    if (!font->Open(ptr, length, false))
+    font = new FTGLPixmapFont(ptr, length);
+    if (false /* !font->Open(ptr, length, false) */ )
       {
       vtkErrorWithObjectMacro(
         tprop,
Index: vtk-5.8.0/Rendering/vtkOpenGLFreeTypeTextMapper.cxx
===================================================================
--- vtk-5.8.0.orig/Rendering/vtkOpenGLFreeTypeTextMapper.cxx	2011-09-01 15:48:10.000000000 +0200
+++ vtk-5.8.0/Rendering/vtkOpenGLFreeTypeTextMapper.cxx	2011-09-01 15:49:47.000000000 +0200
@@ -23,11 +23,11 @@
 #include "vtkToolkits.h"  // for VTK_USE_GL2PS
 
 #include "vtkFreeTypeUtilities.h"
-#include "vtkftglConfig.h"
+//#include "vtkftglConfig.h"
 
 #include "vtkgluPickMatrix.h"
 
-#include "FTFont.h"
+#include <FTGL/ftgl.h>
 
 #ifdef VTK_USE_GL2PS
 #include "vtk_gl2ps.h"
@@ -429,7 +429,7 @@
 
     // Draw the shadow text
 
-    shadow_font->render(this->Input, ftgl_context);
+    //shadow_font->Render(this->Input, ftgl_context);
 
     // Get the font again, Duh, since it may have been freed from the
     // cache by the shadow font
@@ -466,7 +466,7 @@
 
   // Display a string
 
-  font->render(this->Input, ftgl_context);
+  //font->render(this->Input, ftgl_context);
 
   glFlush();
 
