Description: enable some more plugins, disabled by default

--- a/Plugins/CMakeLists.txt
+++ b/Plugins/CMakeLists.txt
@@ -12,7 +12,7 @@
   ENDIF(PARAVIEW_BUILD_PLUGIN_${name})
 ENDMACRO(paraview_build_optional_plugin)
 
-paraview_build_optional_plugin(EyeDomeLighting "EyeDomeLighting" EyeDomeLighting OFF)
+paraview_build_optional_plugin(EyeDomeLighting "EyeDomeLighting" EyeDomeLighting ON)
 paraview_build_optional_plugin(Manta "Manta ray traced rendering" Manta OFF)
 paraview_build_optional_plugin(Moments "Filters for Flux and Circulation Fields" Moments ON)
 paraview_build_optional_plugin(NetDMFReader "NetDMF file format reader" NetDMFReader OFF)
@@ -29,7 +29,7 @@
 paraview_build_optional_plugin(VRPNPlugin "Virtual Reality Private Network (VRPN) tracker" VRPN OFF)
 paraview_build_optional_plugin(VRUIPlugin "Virtual Reality User Interface (VRUI) tracker" VRUI OFF)
 paraview_build_optional_plugin(H5PartReader "Reader for *.h5part files" H5PartReader ON)
-paraview_build_optional_plugin(AdiosReader "Reader for *.bp files based on Adios" AdiosReader OFF)
+paraview_build_optional_plugin(AdiosReader "Reader for *.bp files based on Adios" AdiosReader ON)
 IF(PARAVIEW_BUILD_QT_GUI)
   paraview_build_optional_plugin(CoProcessingScriptGenerator "Plugin for creating python coprocessing scripts" CoProcessingScriptGenerator OFF)
   
